frontend/src/actions/authActions.ts

```diff
@@ -112,15 +113,22 @@ export const userLogOut = () => (dispatch: Dispatch<logoutUserAction | CallHisto
     }
 }
 
+
+
 export const recover = (payload: IUserEmail) =>
   async (dispatch:Dispatch<logoutUserAction | CallHistoryMethodAction>): Promise<void> => {
     const authApiClient = new AuthApiClient()
 
     try {
-      const result = await authApiClient.recover(payload)
-      const {message} = result
-      toast.error(message)
-      dispatch(push('/login'))
+      const {email} = payload
+      if (emailFormatIsValid(email)) {
+        const result = await authApiClient.recover(payload)
+        const {message} = result
+        toast.info(message)
+        dispatch(push('/login'))
```

Comment:
As a user that is not watching his screen 100% time I might miss notification and not be aware if anything was sent to my email - since its just going to sign in page and toast notification dissapears. If possible can you make that after email is sent and recognized on backend, user is shown a message -- like this --

**`Reset your password`**

`An email is on its way to {user.email} with instructions for resetting your password.
If you do not receive the email soon, check that the email address you entered is correct or check your spam folder. `

`Return to SignIn  button`

and if the user clicks this button, he is redirected to SignIn page

Reply:
Done